build-tc

```diff
@@ -2,22 +2,7 @@
 
 set -e
 
-# Installing yarn
-
-YARN_VERSION="1.17.3"
-YARN_LOCATION="$(pwd)/tools/${YARN_VERSION}"
-
-if [ ! -d "$YARN_LOCATION" ]; then
-	mkdir -p ${YARN_LOCATION}
-	cd ${YARN_LOCATION}/
-	wget -qO- https://github.com/yarnpkg/yarn/releases/download/v${YARN_VERSION}/yarn-v${YARN_VERSION}.tar.gz | tar zvx
-	cd ../..
-fi
-
-export PATH="$PATH:$YARN_LOCATION/yarn-v$YARN_VERSION/bin"
```

Comment:
I'd be surprised if the `yarn` commands further down this script are actually running the committed version (and not some random version that happens to be on the build agent)... I'd probably keep this `PATH` line and set the location to be the committed one.

Reply:
According to https://classic.yarnpkg.com/en/docs/cli/policies/#toc-policies-set-version

> yarn policies set-version offers a simple way to check in your Yarn release within your repository. Once you run it, your configuration will be updated in such a way that anyone running a Yarn command inside the project will always use the version you set - and this transparently.

Since we do `cd app` before running `yarn`  I think we are inside the project.

Reply:
interesting, is there a way we could prove it? e.g. by running `yarn -v` and `type -a yarn` in the build script

Reply:
TC build logs already indicates correct version (unless it is a coincidence agent has the same version):

![image](https://user-images.githubusercontent.com/13835317/93063727-954e6600-f66e-11ea-8802-fcb488b52b80.png)

Reply:
It looks up location of yarn on the basis `yarn-path` in `.yarnrc`

```
yarn-path ".yarn/releases/yarn-1.22.5.js"
```

Reply:
If we wanted to be rally sure we could do in `build-tc` script something like 

```
cd app
alias yarn ='node ../.yarn/releases/yarn-*.js'
```

Reply:
any version of yarn <1 should actually use the file referenced in `.yarnrc` if it finds it